strongbox-security/strongbox-authentication-providers/strongbox-ldap-authentication-provider/src/test/resources/strongbox.ldif

```diff
@@ -24,6 +24,19 @@ givenName: Martin
 surname: Todorov
 userPassword: {bcrypt}$2a$10$lpwlxyjvXKzN1ccCrw2PBuZx.eVesWbfmTbsrCboMU.gsNWVcZWMi
 
+# Create users with base64 encoded password of uid mtodorov
+dn: uid=base64encoded-issue-1840,ou=Users,dc=carlspring,dc=com
+objectClass: inetOrgPerson
+objectClass: organizationalPerson
+objectClass: person
+objectClass: top
+uid: base64encoded-issue-1840
+cn: base64encoded-issue-1840
+mail: mtodorov@carlspring.com
+givenName: Martin
+surname: Todorov
+userPassword: {bcrypt}JDJhJDEwJGxwd2x4eWp2WEt6TjFjY0NydzJQQnVaeC5lVmVzV2JmbVRic3JDYm9NVS5nc05XVmNaV01p
```

Comment:
Could you please change this to:
```suggestion
userPassword: {MD5}X03MO1qnZdYdgyfeuILPmQ==
```
I'm sorry - I noticed this after our chat.  
OpenLDAP doesn't support `bcrypt` logins and for this specific case `MD5` is easier to debug via ApacheDS Studio. :)